packages/camera/camera_platform_interface/lib/src/method_channel/method_channel_camera.dart

```diff
@@ -481,6 +481,17 @@ class MethodChannelCamera extends CameraPlatform {
     );
   }
 
+  @override
+  Future<void> setDescriptionWhileRecording(
```

Comment:
This is missing tests. They should go in https://github.com/flutter/plugins/blob/main/packages/camera/camera_platform_interface/test/method_channel/method_channel_camera_test.dart